README.md

```diff
@@ -26,21 +26,25 @@ This example has a backend using the `moneykit` Python sdk and another version t
 
 ### Getting started
 
-1. Copy or symlink your `.env` to `create_link/.env`
+1. Copy the `.env` you created above to `create_link/.env`
 
    ```sh
-   ln -s .env create_link/.env
+   cp .env create_link/.env
    ```
 
-2. Launch your preferred backend.
+2. Change directories into `create_link` and launch your preferred backend.
 
    ```sh
-   dcr --rm --service-ports backend-python
+   cd create_link
+
+   docker compose run --rm --service-ports backend-python
    # or
-   dcr --rm --service-ports backend-python-httpx
+   docker compose run --rm --service-ports backend-python-httpx
```

Comment:
I don't have context on the reason for multiple python backends being here, but I think for the purposes of this section it might be simplest to just list one backend service so no one is confused about which one they should be using? (Probably the httpx - last time I tried to run this, that one worked and the other didn't)